test/server/core/card/CardHighlighting.spec.ts

```diff
@@ -0,0 +1,67 @@
+describe('Card highlighting', function() {
+    integration(function(contextRef) {
+        describe('Card highlighting', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        resources: ['armed-to-the-teeth',
+                            'collections-starhopper',
+                            'covert-strength',
+                            'chewbacca#pykesbane',
+                            'battlefield-marine',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                            'moment-of-peace',
+                        ],
+                        hand: ['vanguard-infantry'],
+                        groundArena: ['wampa'],
+                        spaceArena: ['cartel-spacer'],
+                    },
+                    player2: {
+                        groundArena: ['death-trooper'],
+                        spaceArena: ['mercenary-gunship']
+                    }
+                });
+            });
+
+            it('the prompt before an action and after should be different.', function () {
+                const { context } = contextRef;
+                let currentPossibleActions = context.player1.currentPrompt();
```

Comment:
Should always use `const` or `var` instead of `let`, not sure why ts and the linter aren't catching that in some cases

Reply:
will convert to var.

Reply:
oh, huh. I'll take a look at that at some point